src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, DrawGenerator {
```

Comment:
Ensure that the interfaces 'AreaCalculator' and 'DrawGenerator' do not contain conceptually different methods. If 'draw()' and 'getArea()' are not related, they should not be in the same interface.